mex/common/public_api/connector.py

```diff
@@ -40,26 +43,31 @@ class PublicApiConnector(HTTPConnector):  # pragma: no cover
     TIMEOUT = 10
     API_VERSION = "v0"
 
-    def __init__(self, settings: BaseSettings) -> None:
-        """Create a new Pulic API connection.
+    token_provider: str
+    token_payload: SecretStr
```

Comment:
are these class variables ever used or set?

Reply:
nope, good catch. fixed in bc430df